Store/addNewItem/addNewItem.js

```diff
@@ -0,0 +1,71 @@
+const categoryData = [
+  { title: "===선택===", value: "none" },
+  { title: "전체🍳", value: "total" },
+  { title: "라면🍜", value: "ramen" },
+  { title: "간단안주🥠", value: "anju" },
+  { title: "술🍶", value: "soju" },
+];
+
+// 이미지 미리보기
+function handleFiles(input) {
+  if (input.files) {
+    const reader = new FileReader();
+    reader.onload = function (e) {
+      const previewImage = document.getElementById("previewImg");
+      previewImage.src = e.target.result;
+    };
+    // reader가 이미지 읽도록 하기
+    reader.readAsDataURL(input.files[0]);
+  }
+}
+
+// 카테고리 선택 함수
+function selectCategory() {
+  const createdSelect = document.querySelector(".selectCategory");
+  categoryData.forEach((data) => {
+    const createdOption = document.createElement("option");
+    createdOption.textContent = data.title;
+    createdOption.value = data.value;
+    createdSelect.appendChild(createdOption);
+  });
+}
+
+// 메뉴 이름과 태그 저장 함수
+function saveInfo() {
+  const menu = document.querySelector(".input__menu");
+  const tags = document.querySelector(".input__tag");
+
+  let savedMenu;
+  let savedTag;
+
+  menu.onchange = (e) => {
+    savedMenu = e.target.value;
+    localStorage.setItem("name", savedMenu);
+  };
+
+  tags.onchange = (e) => {
+    savedTag = e.target.value;
+
+    let splitTag = savedTag.split(",");
+    localStorage.setItem("tags", JSON.stringify(splitTag));
+  };
```

Comment:
따로 변수에 할당할 필요 없이 요렇게 해두 되지 않아?!
그리고 addEventListener가 아니라 onchange를 사용한 이유도 따로 있는지 궁금하당!!ㅎㅎ

```suggestion
  menu.addEventListener("change", () => {
    localStorage.setItem("name", menu.value);
  });

  tags.addEventListener("change", () => {
    const splitTag = tags.value.split(",");
    localStorage.setItem("tags", JSON.stringify(splitTag));
  });
```